src/Event/EventRepository.php

```diff
@@ -45,4 +45,12 @@ public function findBySlug(string $eventSlug): ?Event
     {
         return $this->findOneBy(['slug' => $eventSlug]);
     }
+
+    /**
+     * @return Event[]
+     */
+    public function findByApiSecret(string $apiSecret): array
```

Comment:
nebylo by lepší vracet vždy jen jeden Event, nebo null, když není nalezený? Víc Events se stejným api secretem stejně nechceme.

Zjednoduší se ukládání do request attributu, všechny metody, co to kontrolují i ten array_filter

Reply:
to si ale vědomě vytváříš bug.. zjednodušilo by to kód, ale pokud to bude chtít kdokoliv provozovat po nás, tak tam prostě najednou existuje bug, že když má víc eventů jeden auth key, tak to autorizuje jen pro ten první.

Reply:
Chápu připomínku, ale tomu můžeme zabránit třeba tak, že přidáme `unique` na sloupec `api_secret` a potom se to nemůže stát. 

Obrácená situace - dejme tomu, že bychom to udělali s array. Když by tam dorazil auth code pro víc akcí (v nejhorším případě pro všechny stejný), tak bys mohl autorizovat účastníka pro jinou akci, než bylo zamýšleno; to spíš nechceme.